packages/design-system/src/styles/components/_Choice.scss

```diff
@@ -133,20 +144,6 @@ $ds-c-inset-border-width: $spacer-half;
   border-radius: 100%;
 }
 
-/*
-Right-to-Left
-*/
-
-.ds-c-choice--right + label {
-  padding-left: 0;
-  padding-right: $choice-size + $spacer-1;
-
-  &::before {
-    left: auto;
-    right: 0;
-  }
-}
-
```

Comment:
Removing instances of `.ds-c-choice--right` because we no longer support the right to left alignment of choice components and I think that should have been removed previously.

See these commits:
https://github.com/CMSgov/design-system/commit/680304ee58f908d1791842e7bd5e5d92076fae88
https://github.com/CMSgov/design-system/commit/f49673b5b718c9fc1395d1cb3f5ee3f6d441b173